rest/oidc_api.go

```diff
@@ -43,7 +43,7 @@ const (
 var (
 	// ErrNoStateCookie is returned by handler's handleOIDCCallback method when
 	// the state cookie is not found during OpenID Connect Auth callback.
-	ErrNoStateCookie = base.HTTPErrorf(http.StatusBadRequest, "OIDC Auth Failure: No state cookie found")
+	ErrNoStateCookie = base.HTTPErrorf(http.StatusBadRequest, "OIDC Auth Failure: No state cookie found, client needs to support cookies when OIDC callback state is enabled")
```

Comment:
As discussed, maybe also mention the option of the config option to disable state checking if somebody is unable to change how the client behaves.

Reply:
I discussed w/ Sarath, decided to leave out the explicit reference to the config option since this ends up as an end-user error, and end-users don't have any config options on their end.